google/resource_source_repos_repository.go

```diff
@@ -0,0 +1,108 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/sourcerepo/v1"
+	"log"
+)
+
+func resourceSourceReposRepository() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSourceReposRepositoryCreate,
+		Read:   resourceSourceReposRepositoryRead,
+		Delete: resourceSourceReposRepositoryDelete,
+		//Update: not supported,
+
+		Schema: map[string]*schema.Schema{
```

Comment:
Consider adding the `size` as a Computed field. I can see the value for a user who might want to add an output block to their Terraform config to print out the current repo size.